### PR TITLE
A few fixes

### PR DESCRIPTION
**Pull request by Valdnet**

### CODE REVIEW

File: user_manual/pim/calendar.rst [hunk 1]
```diff
@@ -53,27 +49,31 @@ previous calendar, creating a new calendar is the way you should go.
...
             :scale: 50%
 
-1. Click on the three-dot-menu of the respective calendar.
+3. Click on ``Edit color``:
 
 .. figure:: images/calendar_editing.png
             :scale: 50%
 
-2. Click on ``Edit``, ``Download`` or ``Delete``.
+.. note:: Please keep in mind that deleting a calendar is a irreversible action.
```
Comment (tflidd) [inline user_manual/pim/calendar.rst:74]: a**n** irreversible ?

**Review by tflidd — CHANGES_REQUESTED** [review]

File: user_manual/pim/calendar.rst (hunk 1 above)
Comment (tflidd) [reply user_manual/pim/calendar.rst:74]: This doesn't make it better. I think you can correct an obvious typo without doing a separate pull request...

Comment (Valdnet) [reply user_manual/pim/calendar.rst]: Typo corrected 😉.

**Review by tflidd — APPROVED** [review]

**Comment by MorrisJobke** [comment]
Was reverted in  #5848

**Pull request merged**